spatial-tracking-explainer.md

```diff
@@ -183,34 +192,65 @@ The `viewer` reference space is also useful when the developer wants to compare
 
-#### Rays
-An `XRRay` object includes both an `origin` and `direction`, both given as `DOMPointReadOnly`s. The `origin` represents a 3D coordinate in space with a `w` component that must be 1, and the `direction` represents a normalized 3D directional vector with a `w` component that must be 0. The `XRRay` also defines a `matrix` which represents the transform from a ray originating at `[0, 0, 0]` and extending down the negative Z axis to the ray described by the `XRRay`'s `origin` and `direction`. This is useful for positioning graphical representations of the ray.
+However, once a pose is initially established for a space (e.g. a `viewer` reference space or an input source's `gripSpace`), pose matrices should continue to be provided even during tracking loss. The `emulatedPosition` attribute of `XRPose` indicates that the position component of the retrieved pose matrix does not represent an actively tracked position. There are a number of reasons this might be the case. For example:
+* A viewer with orientation-only tracking, whose position within a `local` reference space represents neck modeling.
+* A viewer that has temporarily lost positional tracking, whose position within a reference space represents the viewer's last-known position in that space, plus inertial dead reckoning and/or neck modeling to continue providing a position.
+* A motion controller with orientation-only tracking, which is positioned at an assumed position, e.g. by the user's hip.
+* A motion controller that has temporarily lost positional tracking but is still held, whose orientation continues to update at the last-known position relative to the viewer.
+
+This base tracking loss behavior enables developers to then build whatever tracking loss behavior their scenario requires.
+
+For example, it is common in VR experiences to let the world drag along with the user during positional tracking loss, as this can be preferable to halting the experience. By continuing to render the experience when `XRViewerPose.emulatedPosition` is reporting true, this behavior can be obtained easily. In contrast, when building an AR experience where anchoring of rendered objects to the real world is more critical, developers can detect that `XRViewerPose.emulatedPosition` is reporting false and stop rendering the main scene, falling back to an orientation-only warning.
+
+In contrast, many motion controllers have their own inertial tracking that can continue updating orientation while positional tracking is lost. If using a controller primarily to target distant objects, developers may choose to simply ignore `XRPose.emulatedPosition` and continue to point using each frame's updated target ray as the user rotates the controller. If using a controller to do fine operations, such as painting at the controller's tip, developers may instead choose to stop painting when `XRPose.emulatedPosition` becomes false, ensuring that only high-quality paint strokes are drawn.
+
```

Comment:
Maybe add a paragraph about how developers can use the viewer reference space to display an "Establishing tracking" while other reference spaces may still be attempting to retrieve their first poses?  I'm also ok with this happening in a different PR ;)

Reply:
I think it would be good to continue discussing that in a separate issue.  It does seem unfortunate after all this if apps have to jump through those hoops, especially if the behavior will differ across platforms and UAs, though it may be unavoidable.

Reply:
Let's continue this discussion in #674.